### ISSUE TITLE
Installing LibSassBuilder hides SCSS files from the solution explorer in Rider

### ISSUE THREAD

**Issue by MrDach** [johan-v-r/LibSassBuilder#27]
As the title says my SCSS files disappear in the solution explorer of Jetbrains Rider when I install this library.
I can still see the files in Windows Explorer and they also get compiled when building the project.
Uninstalling the library makes the files re-appear.

Do you have any idea why this is happening?

**Comment by johan-v-r** [comment]
I might depend on the file location, as certain folders are hidden by default like `bin` and `logs` - what is your file path?

**Comment by MrDach** [comment]
The file is located in a path like this: "[project root]/Shared/Test.razor.scss".
And the file is visible without LibSassBuilder installed.
I can even have two projects in the same solution - one with the library installed and one without and the SCSS files are hidden only in the one where it is installed.

**Comment by radverg** [comment]
I can confirm this - all scss files invisible in Rider (macOS).

**Comment by johan-v-r** [comment]
Think I have a fix, just want to do some final testing...

For now you can add the below to your `csproj` to show the files:

```xml
    <ItemGroup>
        <None Include="**/*.scss" />
    </ItemGroup>
```

**Comment by MrDach** [comment]
Awesome, thanks! I'll check it out first then after the summer vacation :)

**Comment by johan-v-r** [comment]
This should be resolved in v1.6.4
